pyop2/types/glob.py

```diff
@@ -464,3 +467,6 @@ def duplicate(self):
             dtype=self.dtype,
             name=self.name
         )
+
+    def increment_dat_version(self):
+        pass
```

Comment:
Since `Constant`s can be changed I don't think that this behaviour is right.

Reply:
This is currently correct because Constants aren't currently versioned. Maybe they should be but that's a different PR.